tests/conftest.py

```diff
@@ -40,11 +40,11 @@ def uuid_list():
     return [
         "5ae279acd9824cbfa85042013cfbf8b7",
         "a401d83c7ec0495f82a8da8d9a389f5b",
-        "81b0bea6aedf4817a74d32d706908df2",
+        "4d4d3477c624836b1b3a3729a7de688a",
         "0726fb4a2341477cbfa36ebd830bd8e3",
         "a7776ed12daa449bb0642e9ea1bb4152",
         "6e147ff372f9498abb5749a1210d8e0a",
-        "498cceede5f14fd991778f007b237803",
+        "a401d83c7ec0495f82a8da8d9a389f5b",
```

Comment:
Honestly no idea why these needed to be changed

Reply:
Don't worry about it. I did not write the tests correctly.